### PR TITLE
DL - Adding image support.

### PR DESCRIPTION
**Pull request by douglasrlee**
JIRA: https://legalshield.atlassian.net/browse/AD-671

### CODE REVIEW

File: paper-password-input.html [hunk 1]
```diff
@@ -17,10 +17,9 @@
 			.visibility-icon {
 				color: var(--disabled-text-color);
 				/** Bottom align visibility button with input box */
-				height: 32px;
-				width: 32px;
-				padding: 0 4px;
-				margin-top: -6px;
+				height: 22px;
+				width: 22px;
+				padding: 0;
```
Comment (julamula) [inline paper-password-input.html]: can we alphabetize these?

File: paper-password-input.html [hunk 2]
```diff
@@ -17,10 +17,9 @@
 			.visibility-icon {
```
Comment (kylessnell) [inline paper-password-input.html]: what is going on with the spacing in this file

Comment (douglasrlee) [inline paper-password-input.html]: We forked it and they used tabs..

Comment (kylessnell) [inline paper-password-input.html]: grumpy-cat

**Pull request merged**